### ISSUE TITLE
Change the scope adjustment default keys

### ISSUE THREAD

**Issue by ulteq** [acemod/ACE3#771]
I would to set the arrow keys as default keybind for turning the scope adjustment knobs:
* Elevation - arrow up/down
* Windage - arrow left/right)

1: http://forums.bistudio.com/showthread.php?190433-ACE3-A-collaborative-merger-between-AGM-CSE-and-ACE&p=2923653&viewfull=1#post2923653

2: https://ace3public.slack.com/archives/public/p1429459767002402

**Comment by KoffeinFlummi** [comment]
Opposed for two reasons:

- The vanilla keys for zeroing are <kbd>Page Up</kbd> and <kbd>Page Down</kbd>, meaning that pretty much all Arma 3 players, even those who played Arma 2 are used to those by now.

- The arrow keys are often used (especially be left-handed people) as movement keys, meaning that out-of-the-box ACE would block an essential part of regular gameplay, which is not as likely when using more "obscure" key bindings such as the <kbd>Page {Up,Down}</kbd>.

**Comment by esteldunedain** [comment]
I agree with Flummi, and people who use adjustments very often could easily rebind those keys if they want to (and probably will).

TL;DR: Keep it close to vanilla for all players. Let people rebind them as they see fit.

**Comment by ulteq** [comment]
>The arrow keys are often used (especially be left-handed people) as movement keys, meaning that >out-of-the-box ACE would block an essential part of regular gameplay, which is not as likely when >using more "obscure" key bindings such as the Page {Up,Down}.
I am convinced.
